pkg/api/util.go

```diff
@@ -6,7 +6,9 @@ import (
 )
 
 const (
-	unixPrefix = "unix:"
+	unixPrefix  = "unix:"
```

Comment:
Isn't `unix:///path/to/socket.sock` the commonly used socket scheme (not sure though)?

Reply:
Not sure exactly. But it worked like this before 🤷